hathor/consensus/consensus.py

```diff
@@ -75,11 +76,13 @@ def update(self, base: BaseTransaction) -> None:
         assert meta.validation.is_valid()
         try:
             self._unsafe_update(base)
-        except Exception:
-            meta.add_voided_by(self._settings.CONSENSUS_FAIL_ID)
-            assert base.storage is not None
-            base.storage.save_transaction(base, only_metadata=True)
-            raise
+        except BaseException:
```

Comment:
I think we should keep handling `Exception`. Even if it is a bug in the code, the transaction should be marked as fail.

Reply:
Sure, I brought back that transaction marking